python/lsst/pipe/tasks/quickFrameMeasurement.py

```diff
@@ -227,6 +237,42 @@ def _calcMedianXxYy(objData):
         medianYy = np.nanmedian([element['xx'] for element in objData.values()])
         return medianXx, medianYy
 
+    @staticmethod
+    def _getCentreOfMass(exp, nominalCentroid, boxSize):
```

Comment:
Don't we have to call this `_getCenterOfMass` ?

Reply:
Oh yeah, probably.